hooks/026-configure-plymouth.chroot

```diff
@@ -0,0 +1,109 @@
+#! /bin/sh -ex
+
+# Note that we have overrides for plymouth services in:
+# static/usr/lib/systemd/system/plymouth-start.service.d/core-override.conf
+# static/usr/lib/systemd/system/plymouth-quit.service.d/core-override.conf
+# static/usr/lib/systemd/system/plymouth-halt.service.d/core-override.conf
+# static/usr/lib/systemd/system/plymouth-reboot.service.d/core-override.conf
+# static/usr/lib/systemd/system/plymouth-kexec.service.d/core-override.conf
+# static/usr/lib/systemd/system/plymouth-poweroff.service.d/core-override.conf
+
+cat << 'EOF' > /lib/systemd/system/splash-client.service
+[Unit]
+Description=Update Plymouth Boot Screen
+After=plymouth-start.service snapd.service snapd.socket
+ConditionKernelCommandLine=snapd_recovery_mode=run
+
+[Service]
+ExecStart=-/usr/bin/splash-client
+EOF
+
+mkdir -p /etc/plymouth
+cat << 'EOF' > /etc/plymouth/plymouthd.conf
+[Daemon]
+Theme=vendor
+ThemeDir=/run/mnt/gadget/splash/plymouth/themes
+ShowDelay=0
+DeviceTimeout=8
+EOF
+
+mkdir -p /usr/share/plymouth
+cat << 'EOF' > /usr/share/plymouth/plymouthd.defaults
+[Daemon]
+Theme=ubuntu-core
+ShowDelay=0
+DeviceTimeout=8
+EOF
+
+cat << 'EOF' > /lib/systemd/system/plymouth-show-run.service
+[Unit]
+Description=Display state on splash
+After=plymouth-start.service
+ConditionKernelCommandLine=snapd_recovery_mode=run
+ConditionPathExists=/run/plymouth/pid
+
+[Service]
+ExecStart=-/bin/plymouth display-message --text='Starting system...'
+Type=oneshot
+TimeoutSec=20
+
+[Install]
+WantedBy=sysinit.target
```

Comment:
I am not sure why there is an `[Install]` section if it there is `Requires` added in `plymouth-start.service`